html/pages/device/graphs.inc.php

```diff
@@ -22,11 +22,7 @@
     }
 }
 
-// These are standard graphs we should have for all systems
-$graph_enable['poller']['poller_perf'] = 'device_poller_perf';
-if (can_ping_device($attribs) === true) {
-    $graph_enable['poller']['ping_perf']   = 'device_ping_perf';
-}
+enable_graphs($device, $graph_enable);
```

Comment:
How come the poller_perf and ping_perf graphs have been removed from $graph_enable?